core/cli/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ add_library(cli INTERFACE)
 target_link_libraries(cli INTERFACE
     Boost::program_options
     fmt::fmt
+    file
```

Comment:
move this to [`target_link_libraries(fuhon-node-cli`](https://github.com/filecoin-project/cpp-filecoin/blob/master/core/cli/node/CMakeLists.txt#L10)

Reply:
Fixed